HIP-Basic/warp_shuffle/README.md

```diff
@@ -0,0 +1,49 @@
+# HIP-Basic Warp Shuffle Example
+
+## Description
+Kernel code is SIMT-executed by groups of threads within the same block, known as _warps_ (NVIDIA) or _wavefronts_ (AMD). Each block is divided into as many warps as the block's size allows, if the block size is less than the warp size then part of the warp just stays idle (as happens in this example). 
+In NVIDIA GPUs the warp size is 32 threads, while in AMDs it is 64 threads for architectures previous to RDNA1. After RDNA1 all architectures support both 32 and 64 warp sizes.
```

Comment:
We have guidance for referring to RDNA, we need make sure this conforms to it. I believe there needs to be TM the first time.

Reply:
Added